Marsiple/Common/Extensions/UIBarButtonItem-Marsiple.swift

```diff
@@ -10,7 +10,10 @@ import UIKit
 
 extension UIBarButtonItem {
     static func createCommentItem(target: Any, action: Selector) -> UIBarButtonItem {
```

Comment:
We could eject the `create` from the method name, as it's clear from the method signature what it does. 
What do you think?

Reply:
Yes, I would say it's clear what it does.